build/Targets/References.targets

```diff
@@ -222,7 +222,7 @@
       <ItemGroup>
 
         <PackageReference Include="Moq" Version="4.2.1402.2112" />
-        <PackageReference Include="xunit" Version="2.1.0" />
+        <PackageReference Include="xunit" Version="2.2.0-beta4-build3444" />
```

Comment:
This is the version that [roslyn is using](https://github.com/dotnet/roslyn/blob/2223e15607ec068e17c23dc6aa9718a8366dcb80/build/Targets/Packages.props#L223)  and therefore the version required by the vsi test framework. I could try amd make it so that only the VSI tests depend on this version but it becomes difficult to reason about which package nuget deploys as it tries to unify things.